resources_pages/install_ds_stack_windows.md

```diff
@@ -3,290 +3,540 @@ layout: page
+`conda` will show you the packages that will be downloaded,
+and you can press enter to proceed with the installation.
+If you want to answer `yes` by default and skip this confirmation step,
+you can replace `conda install` with `conda install -y`.
 
-We will be using PostgreSQL as our database management system. You can download it from [here](https://www.enterprisedb.com/thank-you-downloading-postgresql?anid=1256714). Follow the instructions for the installation. In the password page, type whatever password you want, but make sure you'll remember it later. For all the other options, use the default. You can execute `SQL Shell` to test if you were successful.
+> Note: we will use many more packages than those listed above across the MDS program, however we will manage these using virtual environments (which you will learn about in DSCI 521: Platforms for Data Science).
 
-## Visual Studio Code
+### JupyterLab setup
 
-### Installing
+We will be using the Jupytext Python package and the JupyterLab git extension to facilitate using Jupyter notebooks with Git & GitHub. Install them via the following commands:
 
-We need a text editor to be able to write complete applications. One is available through Jupyter, but sometimes it is helpful to have a standalone text editor, for this we will be using the open-source text editor Visual Studio Code (VS Code). You can download VS Code at https://code.visualstudio.com/download. At "Select Additional Tasks" make sure that "Add to path" is selected (I also like to select all the other three items - see figure below).
+```
+conda install nodejs=10.*
+pip install --upgrade jupyterlab-git
+conda install jupytext=1.*
+jupyter lab build
+```
 
-![](/resources_pages/imgs/vs_code.png)
+To test that your JupyerLab installation is functional, you can type `jupyter lab` into a terminal, which should open a new tab in your default browser with the JupyterLab interface.
+
+![](/resources_pages/imgs/jupyter_lab.PNG)
 
-You can test that VS code is installed and can be opened from the Powershell by running the following command:
+## R, IRkernel, Rtools, and RStudio
+
+R is another programming language that we will be using a lot in the MDS program. We will use R both in Jupyter notebooks and in RStudio.
+
+### R
+
+Go to <https://cran.r-project.org/bin/windows/base/> and download the latest version of R for Windows (4.0.2 at the time of writing). Open the file and follow the installer instructions accepting the default configuration.
+
+After the installation is complete, we will add the R executables to the PATH variable in terminal so that your can use it without typing the full path to R each time. Open a terminal and type:
 
 ```
-code --version
+code ~/.bash_profile
 ```
 
-you should see something like this if you were successful:
+Append the following line to the file
+
 ```
-1.36.1
-2213894ea0415ee8c85c5eea0d0ff81ecc191529
-x64
+# Add R and Rscript to PATH
+export PATH="/c/Program Files/R/R-4.0.2/bin/x64":$PATH
```

Comment:
I don't know if they will be using R from terminal other than below for the setup, but thought it would make it sense to enable it so that students can see how it is done and use if needed.